### ISSUE TITLE
Installation completed, now what?

### ISSUE THREAD

**Issue by giveen** [DefectDojo/godojo#27]
Apologizes for sounding like an idiot, but I have completed the installation successfully, now what?

How do I launch it? How do I go to the internal website? I went to localhost:8080 and nothing.
![1](https://user-images.githubusercontent.com/1180939/134723016-6cbce794-058a-4f75-a412-4037fdaa0c0e.PNG)

**Comment by mtesauro** [comment]
Now you have a party :+1: 

Most people will put DefectDojo behind some sort of reverse proxy. There's loads of ways to front DefectDojo with Nginx, or Apache or something else so godojo doesn't try to solve that problem.  

However, I did write up some things to start up DefectDojo and stop it so you have something to play with right away.  You can find those in the repo in the "scripts" directory - there's a README there that talks through your post install options. 

https://github.com/DefectDojo/godojo/tree/master/scripts 

Basically, I'd recommend putting your favorite HTTP server setup as a reverse proxy with TLS configured there rather than getting it working with DefectDojo/Django.  Unless you're just testing things out, then connecting to DefectDojo just for eval purposes probably doesn't need TLS and you can just use HTTP.

HTH

**Comment by giveen** [comment]
@mtesauro I appreciate the quick response, I'll take a look at these on Monday.

**Comment by giveen** [comment]
This worked perfectly, though I did have to install the package 'screen', thank you again.
